src/transformers/trainer.py

```diff
@@ -1382,8 +1390,7 @@ def prediction_step(
             Tuple[Optional[float], Optional[torch.Tensor], Optional[torch.Tensor]]:
             A tuple with the loss, logits and labels (each being optional).
         """
-        has_labels = any(inputs.get(k) is not None for k in ["labels", "lm_labels", "masked_lm_labels"])
```

Comment:
No need to test for old deprecated argument names since they have all been changed in the lib and the user can now set their own name if they have an old model they are still using.

Reply:
good! this will reduce cruft